python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -296,7 +298,6 @@ def to_default_asset_events(
                     output_name=dagster_name_fn(event_node_info),
                     metadata={
                         **default_metadata,
-                        "Execution Duration": duration_seconds,
```

Comment:
Do we have a test that ensures that this is still present on `AssetMaterialization` and `Output`?